File: 20260913/mymap.js
myMap = { book: 10, milk: 5 };
console.log(myMap.book); 
console.log(myMap.milk);
console.log(myMap["book"]);
console.log(myMap["milk"]);

myMap.hello = function() {
    console.log("Hello from myMap!");
};
myMap.hello();

File: 0927/my_map.cpp
#include <iostream>
#include <map>

int main() {
    std::map<std::string, int> my_map = {
        {"book", 10},
        {"milk", 5}
    };

    std::cout << my_map.at("book") << std::endl;
    std::cout << my_map.at("milk") << std::endl;

    return 0;
}